playbooks/generic-build_images.yml

```diff
@@ -10,7 +10,7 @@
     #http://jinja.pocoo.org/docs/2.10/templates/#default
     - name: Check if developer mode should be enabled
       set_fact:
-        developer_mode: "{{ (lookup('env','OSH_DEVELOPER_MODE') | default('False', true) ) | bool }}"
+        developer_mode: "{{ (lookup('env','SOCOK8S_DEVELOPER_MODE') | default('False', true) ) | bool }}"
```

Comment:
nit: I think a follow-up patch can change that by loading the common vars, and setting fact for it, to prevent duplication. We could even remove the set_fact if developer_mode is changed appropriately everywhere and loading the common vars. But it's okay for now.

Reply:
make sense